v1/personio.go

```diff
@@ -17,6 +17,8 @@ const DefaultBaseUrl = "https://api.personio.de/v1"
 
 const timeOffsMaxLimit = 200
 
+const QUERY_DATE_FORMAT = "2006-01-02"
```

Comment:
Could you please rename this to `queryDateFormat` to make it a package private constant and comply to the golang naming convention?

Reply:
This has now been done. :)